src/Umbraco.Core/Persistence/Migrations/Upgrades/TargetVersionSevenThreeZero/AddRelationTypeForDocumentOnDelete.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Umbraco.Core.Configuration;
+using Umbraco.Core.Models.Rdbms;
+
+namespace Umbraco.Core.Persistence.Migrations.Upgrades.TargetVersionSevenThreeZero
+{
+    [Migration("7.3.0", 0, GlobalSettings.UmbracoMigrationName)]
+    public class AddRelationTypeForDocumentOnDelete : MigrationBase
+    {
+        public override void Up()
+        {
+            Execute.Code(AddRelationType);
```

Comment:
This is better executed with this code (for future reference):

    Insert.IntoTable("umbracoRelationType").Row(new
    {
        dual = false, 
        parentObjectType = new Guid(Constants.ObjectTypes.Document), 
        childObjectType = new Guid(Constants.ObjectTypes.Document),
        name = Constants.Conventions.RelationTypes.RelateParentDocumentOnDeleteName,
        alias = Constants.Conventions.RelationTypes.RelateParentDocumentOnDeleteAlias
    });